EIPS/eip-7742.md

```diff
@@ -44,37 +44,57 @@ and deployment of changes to the blob count easier.
 
-The target is currently specified as a fixed value in relation to the blob count. The Ethereum community intends to increase
-the blob parameters as part of its scaling strategy and the ability to have a more flexible target value in relation to
-the blob max is desirable to reduce rigidity in this protocol parameter.
+Hence we entirely deprecated the `MAX_BLOB_GAS_PER_BLOCK` checks from [EIP-4844](./eip-4844.md).
 
-Even if the EL keeps a fixed target value based on the max, removing the max implies the EL would not know what the target
-value should be. To address this lack of information, this EIP proposes the CL sends the current target value to the EL
-with each provided payload over the Engine API. The EL block header will also need to be extended with this target value
-to preserve the security of optimistic sync.
+#### Target blobs per block
+
+The target is currently specified as a fixed value in relation to the blob count. The Ethereum community intends to increase the blob parameters as part of its scaling strategy and the ability to have a more flexible target value in relation to the blob max is desirable to reduce rigidity in this protocol parameter.
+
+To this end CL now provides a blob count target as well during block production and the EL block header is extended with this `target_blob_count` value to preserve the security of optimistic sync as well as for the purpose of `excess_blob_gas` and blob base fee calculations.
+
+#### Updated Fee calculations
+
+With the changing target, we also need to scale the `BLOB_BASE_FEE_UPDATE_FRACTION` from [EIP-4844](./eip-4844.md) accordingly to bound the price jumps by +-12.5%. But this introduces some irregularity on the fork block because excess blob gas is effectively scaled down by this changed factor in the fee calculations.
+
+However we consider the target change events to be few and far between and hence not a cause of concern. The likely effect of this factor update will be to bring down the base fee on the fork block and hence would not negatively impact the transactions in txpool.
 
 ## Specification
 
-### Block structure and validity
+### Execution Block structure and validity
 
 Upon activation of this EIP, execution clients **MUST** extend the header schema with an
-additional 64-bit field: the `target_blob_count`. This value is set to the current target blob count. The Engine API
-is modified along with this EIP to provide the `target_blob_count` with each payload and implementations can use this
-value to correctly set the block header field.
+additional 64-bit field: the `target_blob_count`. This values is set to the one provided by the CL with the modified engine api fcU's payload attributes.
```

Comment:
would prefer to keep reference to the Engine API more abstract, e.g. in the event we removed the FCU pattern

could even argue to remove all references to the Engine API (as it is all together an implementation detail)

Reply:
updated